fairlearn/reductions/exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -226,13 +224,13 @@ def _mean_pred(dataX, hs, weights):
                             column_names)
 
 
-def expgrad(dataX, dataA, dataY, learner,
-            constraints=moments.DemographicParity(),
-            eps=0.01,
-            T=50,
-            nu=None,
-            eta_mul=2.0,
-            debug=False):
+def exponentiated_gradient(dataX, dataA, dataY, learner,
+                           constraints=moments.DemographicParity(),
+                           eps=0.01,
```

Comment:
Unrelated to this PR, but maybe in general consider documenting where these default values come from

Reply:
Great idea!

Reply:
I'll ask Miro who came up with these.